tests/test_remote_write.py

```diff
@@ -37,27 +37,19 @@ def setUp(self):
     def test_addr(self):
```

Comment:
Even though this has not changed in this PR, i think a name such as `test_addr` is not specific enough. Some thing like this : "test_address_can_be_set" is more indicative of the nature of the test. Same comment for other tests. An aleternative (though not self-documenting) is to put a comment. It should not be necessary to read the whole test just to figure out what the test is testing.

Reply:
I will fix this